scripts/configure-installer-tests-cluster.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+# Setup to find nessasary data from cluster setup
+## Constants
+HTPASSWD_FILE="./htpass"
+USERPASS="developer"
+HTPASSWD_SECRET="htpasswd-secret"
+# Overrideable information
+AUTH_DIR=${AUTH_DIR:-"$(pwd)/auth"}
```

Comment:
Ig this script is directly tailored for use with `openshift-installer` 

We can use just one variable pointing to assets dir (`--dir` flag for installer).

All the required files are than within this dir.

kubeconfig - `$INSTALLER_ASSETS_DIR/auth/kubeconfig`
admin password - `$INSTALLER_ASSETS_DIR/auth/kubeadmin-password`